services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/FlowCrudSpec.groovy

```diff
@@ -280,99 +280,37 @@ class FlowCrudSpec extends BaseSpecification {
-                    dominantFlow.destination.vlanId = 0
-                },
-                getError: { FlowPayload flowToError ->
-                    portError(flowToError.destination.portNumber, flowToError.destination.datapath, flowToError.id)
-                }
-            ],
-            [
-                conflict: "vlan vs no vlan on same port on src",
-                makeFlowsConflicting: { FlowPayload dominantFlow, FlowPayload flowToConflict ->
-                    flowToConflict.source.portNumber = dominantFlow.source.portNumber
-                    dominantFlow.source.vlanId = 0
-                },
-                getError: { FlowPayload flowToError ->
-                    portError(flowToError.source.portNumber, flowToError.source.datapath, flowToError.id)
-                }
-            ],
-            [
-                conflict: "no vlans both flows on same port on src",
-                makeFlowsConflicting: { FlowPayload dominantFlow, FlowPayload flowToConflict ->
-                    flowToConflict.source.portNumber = dominantFlow.source.portNumber
-                    flowToConflict.source.vlanId = 0
-                    dominantFlow.source.vlanId = 0
-                },
-                getError: { FlowPayload flowToError ->
-                    portError(flowToError.source.portNumber, flowToError.source.datapath, flowToError.id)
-                }
-            ],
-            [
-                conflict: "no vlans both flows on same port on dst",
-                makeFlowsConflicting: { FlowPayload dominantFlow, FlowPayload flowToConflict ->
-                    flowToConflict.destination.portNumber = dominantFlow.destination.portNumber
-                    flowToConflict.destination.vlanId = 0
-                    dominantFlow.destination.vlanId = 0
-                },
-                getError: { FlowPayload flowToError ->
-                    portError(flowToError.destination.portNumber, flowToError.destination.datapath, flowToError.id)
-                }
-            ],
-            [
-                conflict: "the same flowId",
-                makeFlowsConflicting: { FlowPayload dominantFlow, FlowPayload flowToConflict ->
-                    flowToConflict.id = dominantFlow.id
-                },
-                getError: { FlowPayload flowToError ->
-                    "Could not create flow: Flow $flowToError.id already exists"
-                }
-            ]
-        ]
+        where:
+        data << getConflictingData()
+    }
+
+    @Unroll("Unable to update flow with #data.conflict")
+    def "Unable to update flow with conflicting vlans or flow IDs"() {
+        given: "Two potential flows"
+        def (Switch srcSwitch, Switch dstSwitch) = topology.activeSwitches
+        def flow1 = flowHelper.randomFlow(srcSwitch, dstSwitch)
+        def flow2 = flowHelper.randomFlow(srcSwitch, dstSwitch)
+
+        def conflictingFlow2 = flowHelper.randomFlow(srcSwitch, dstSwitch)
```

Comment:
If possible, please rename conflictingFlow2 to a "conflictingFlow". It is a bit confusing now.

Reply:
Done.